### DISCUSSION TITLE
since Quasar 2.16.0 lang ext changed

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar#17173] [General - Components / Directives / etc] [answered]
As mentioned yet here on discord:
https://discord.com/channels/415874313728688138/806172040804827188/1237145369738084415
the lang extension changed from `mjs` to `js`

So this might be a breaking change, if one uses this Q lang packs in his i18n boot file as described there.

**Comment by zanzara** [comment]
Can anyone confirm this change of extension?
Or is there smt more to do when upgrading to 2.16?

**Reply by rstoenescu** [reply, marked as answer]
Yes, hard-coded extension should be switched to `.js`. It's in the release notes as well.
